File: searching-algorithms/linear_search.py
def linearSearch(a, searchElement):
    arraySize = len(a)
    for i in range(0, arraySize):
        if(a[i] == searchElement):
            return 1
    return -1

a = [1, 4, 2, 5, 3]
searchElement = 5
result = linearSearch(a, searchElement)
print("Element is not present in array") if(result == -1) else  print("Element is present in array")

File: searching-algorithms/linear_search.c
#include<stdio.h>

int linearSearch(int a[], int searchElement, int arraySize);

int linearSearch(int a[], int searchElement, int arraySize){
	int i, flag = 0;
	for(i=0; i<= arraySize-1; i++){
		if(searchElement == a[i]){
			flag = 1;
		}
	}
	if(flag == 1){
		return 1;
	}
	else{
		return -1;
	}
}
void main(){
	int a[] = {3, 5, 1, 4, 2};
	int arraySize = sizeof(a)/sizeof(a[0]);
	int searchElement = 1; 
	int result = linearSearch(a, searchElement, arraySize);
	(result == 1) ? printf("Element is present in the array"): printf("Element is not present in the array"); 
}
